grim.go

```diff
@@ -146,6 +151,11 @@ func (i *Instance) BuildNextInGrimQueue(logger *log.Logger) error {
 	return nil
 }
 
+func repoIsAlive(hook *hookEvent) bool {
+	url := hook.URL
+	return len(url) >= 12 && url[len(url)-12:] != "000000000000"
```

Comment:
The push event does have a deleted(true/false), after some trials, it can represent if a branch get deleted or not